### DISCUSSION TITLE
Failed to sync and upload to s3 bucket

### DISCUSSION THREAD

**Discussion by lzhoujc** [aws/aws-cli#7152] [Q&A] [answered]
I have this script where I'm syncing a local directory to s3, but it's only migrating the `.log` files to the s3 and not the `.gz` files. 

```
aws s3 sync local_directory s3://bucket-name   --exclude "*log" --include "-*.gz"
```

A warning is also splashed on my terminal that reads:

```
warning: Skipping file ... File/Directory is not readble.
```

What is wrong with this script that I'm not seeing? Should I be using a `cp` instead of a `sync`?

**Comment by aBurmeseDev** [comment, marked as answer]
Hi @lzhoujc, thanks for reaching out.

I noticed dash/hyphen inside the --include value in your command, is that intentional? I would double check for any typos as mentioned in [this example](https://github.com/aws/aws-cli/issues/5614).

Also, I'm not sure what your local directory path looks like but here's another possible root cause where you'd have to include the path in the --include value.

If you're still having issue, can you share the debug log by adding `--debug` to the end of the command like this `aws s3 sync local_directory s3://bucket-name --exclude "*" --include "*" --debug`?

Please let me know if any of this resolves the issue.

**Comment by evilcrash** [comment]
how are the permissions set on the files?
